contribs/gmf/src/directives/layertree.js

```diff
@@ -975,16 +977,19 @@ gmf.LayertreeController.prototype.zoomToResolution = function(treeCtrl) {
 /**
  * Collect and return all ids of this layer node and all child nodes as well.
  * @param {GmfThemesNode} node Layer tree node.
+ * @param {boolean=} opt_editable Whether the node needs to be editable to
+ *     have its id returned.
  * @return {Array.<number|string>} Layer names.
  */
-gmf.LayertreeController.getLayerNodeIds = function(node) {
+gmf.LayertreeController.getLayerNodeIds = function(node, opt_editable) {
+  var editable = opt_editable === true;
   var ids = [];
   var children = node.children || node;
   if (children && children.length) {
     children.forEach(function(childNode) {
       ids = ids.concat(gmf.LayertreeController.getLayerNodeIds(childNode));
```

Comment:
I don't understand.  Isn't that already the case ?  If a child node has no children, then we go in `else if` below and they are managed, right ?